metrics_api/lib/opentelemetry/metrics/meter.rb

```diff
@@ -43,8 +43,8 @@ def initialize
       # @param description [optional String] an optional free-form text provided by user.
       #
       # @return [nil] after creation of counter, it will be stored in instrument_registry
-      def create_counter(name, unit: nil, description: nil)
-        create_instrument(:counter, name, unit, description, nil) { COUNTER }
+      def create_counter(name, unit: nil, description: nil, **advisory_parameters)
```

Comment:
Could you add some documentation to each of the instruments about the relevant `**advisory_parameters` named keyword args?